### DISCUSSION TITLE
Cannot find the crt file for a profile? (chromeOS onc generation)

### DISCUSSION THREAD

**Discussion by leafgrass95** [pivpn/pivpn#1538] [Support]
Hello, I am a chromeOS user who cannot run android apps, therefore I am reliant on the inbuilt problematic ChromeOS VPN client (that for some random reason doesn't support OVPN because they are insane).

To generate the required ONC file I need the CRT for the vpn profile to generate the client key, client pk12 and ca.crt, following the instructions here https://github.com/pivpn/pivpn/issues/27#issuecomment-217579913 I need to find the crt file generated by PiVPN, but I can't find it in the directory, can anyone tell me where they have moved to?

**Comment by coolapso** [comment]
your description is a bit confusing, not exactly sure what certificates you are looking for, but usually what you need is the content of the certificates generated by `pivpn add` and those  should be in the home directory of the user you specified during pivpn instalation, on that certificate file you should have everything you need. 

Moving to discussions.
